fujiji-client/src/server/api/uploadImage.js

```diff
@@ -0,0 +1,31 @@
+import axios from 'axios';
+import FormData from 'form-data';
+
+export default async function uploadImage(image) {
+  try {
+    const data = new FormData();
+    data.append('image', image, image.name);
+    const result = await axios.post(
+      'https://fujijifunctions.azurewebsites.net/api/uploadImage?code=EzRnmAilmvcIaVoglBsGqoXiW14jyhoaqCGoZNdbypwpCmEr4QShGw==',
```

Comment:
Is this a secret?
`code=EzRnmAilmvcIaVoglBsGqoXiW14jyhoaqCGoZNdbypwpCmEr4QShGw=='`

We might want to move this to environment variables if that's the case.

Reply:
That is the address of the function

Reply:
Sorry, I mean the content of the query param `code=` 

If that one is also not a secret, it looks good then!

Reply:
no its just the unique reference to the function's code, so yeah its good